### ISSUE TITLE
motion control broken since nightly-1803

### ISSUE THREAD

**Issue by ZungBang** [citra-emu/citra#6178]
Motion control (gyro) works for me in nightly-1802 (a93d7a8d3ab02e3ba72013cbf92d5cae84d555d3) and stopped working in nightly-1803 (1ddea27ac8fbc2240b961016ec0f6ff20a6ccd6a).
Tested in several games, with a sony dualsense 5 controller, on debian/testing.
Citra installed via flatpak.
The attached log files are from two sessions (one for each version) where I first auto map the controller, then setup the motion provider as sdl, launch a game (fluidity spin-cycle), verify that motion either works or doesn't and then quit with alt-f4.
[citra_log.nightly.1802.txt](https://github.com/citra-emu/citra/files/9954839/citra_log.nightly.1802.txt)
[citra_log.nightly.1803.txt](https://github.com/citra-emu/citra/files/9954841/citra_log.nightly.1803.txt)

**Comment by poudink** [comment]
what about the non-flatpak version? could this be a problem with flatpak permissions?

**Comment by ZungBang** [comment]
Not sure what you mean by permissions here, but I just checked with the non-flatpak releases for linux, and got the same behavior.

**Comment by GPUCode** [comment]
Looked at the commit multiple times and I couldn't spot any change that would break motion control specifically. I'll try looking again

**Comment by ZungBang** [comment]
I re-tested with the linux (non-flapak) releases, following these steps:

1. remove ~/.config/citra-emu/qt-config.ini
2. launch citra-qt
3. auto-map controller buttons (via Emulation->Configure->Controls->Auto Map)
4. select motion provider (via Emulation->Configure->Controls->Motion/Touch->Motion Provider, select SDL, press Configure)
5. lauch game via File->Load File
6. check motion control

I get the same bahavior, namely motion control works w/ nightly-1802, and doesn't work w/ nightly-1803.

The resulting qt-config.ini **is** different. I've attached the output of diff. Note that it seems that not only motion is not setup correctly but also axis polarity is wrong for zl and zr buttons.

[diff.txt](https://github.com/citra-emu/citra/files/9964007/diff.txt)

**Comment by ZungBang** [comment]
OK, I skipped the controller setup step and used the qt-config.ini that was generated by nightly-1802 with nightly-1803, and this fixes motion control. As soon as I re-selected the motion provider, motion control stopped working.

It just so happens that I had to configure a new dualsense 5 controller and hit this problem.

So, it's actually motion control configuration that is broken since nightly-1803 and not motion control per-se.
